packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
+      }
+      if (etiquetas[i].hasAttribute('src')) {
+        try {
+          fetch(etiquetas[i].getAttribute('src'))
+            .then((res) => res.text())
+            .then((data) => {
+              let codigoCom = esJscore.compile(data)
+              let elemento = document.createElement('script')
+              elemento.innerHTML = codigoCom
+              if (
+                etiquetas[i].hasAttribute('mode') &&
+                etiquetas[i].getAttribute('mode') == 'modulo'
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)